lib/renaissance/auctions/auction.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Renaissance.Auction do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @required_fields ~w(title description user_id price end_date)a
+  @optional_fields ~w()a
+
+  schema "auctions" do
+    field :title, :string
+    field :description, :string
+    field :price, Money.Ecto.Amount.Type
+    field :end_date, :utc_datetime
+    belongs_to :user, User
```

Comment:
Getting this warning here:

> warning: invalid association `user` in schema Renaissance.Auction: associated schema User does not exist
  lib/renaissance/auctions/auction.ex:1: Renaissance.Auction (module)